src/Steps/Release/PublishRelease.php

```diff
@@ -253,12 +256,11 @@ protected function updateGithubChangelog(OutputInterface $output, LibraryRelease
             'prerelease' => !$version->isStable(),
             'draft' => false,
         ];
-        $changelog = $release->getChangelog();
-        if ($changelog) {
-            $releaseData['body'] = $changelog;
+        $notes = $this->getReleaseNotes($output, $client, $release);
+        if ($notes) {
+            $releaseData['body'] = $notes;
         }
```

Comment:
```suggestion
        $releaseData['body'] = $this->getReleaseNotes($output, $client, $release)
```
We're throwing a hard exception in `getReleaseNotes()` if it fails

Reply:
I don't know for certain if the API will return an empty string if there were no changes but I suspect it would. In that case the conditional statement is still valid.